R/pkg/R/generics.R

```diff
@@ -1376,6 +1376,10 @@ setGeneric("spark.kstest", function(data, ...) { standardGeneric("spark.kstest")
 #' @export
 setGeneric("spark.lda", function(data, ...) { standardGeneric("spark.lda") })
 
+#' @rdname spark.linearSvc
+#' @export
+setGeneric("spark.linearSvc", function(data, formula, ...) { standardGeneric("spark.linearSvc") })
```

Comment:
svm with linear kernel in glmnet package has the same functionality as this one. But they support other kernels. `linearSvm` is better than `linearSvc` (c denotes classifier)?